database/nakadi/10_data/00_create_schema.sql

```diff
@@ -0,0 +1,18 @@
+CREATE ROLE zalando_nakadi_data_owner;
```

Comment:
If you mean zalando-db-commons package -- I wouldn't say it's more clean...
It was my decision not to use it and not to use code from it.
First of all, it brings a new dependency. Dependency itself is not a problem, but the package is not open-source and this is already a problem.
Also this package creates a lot of unneeded (for us) schemas, roles and functions. Some of these roles are obsolete. Do we really need to have all overhead for a small database where we don't event have a sprocs?